### PR TITLE
Update Readme file for better understanding.

### PR DESCRIPTION
**Pull request by exit-scammed**
Preferrably the English language. You can retranslate Chinese lines for CN users, of course.
Feel free to make any change/review.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,41 +1,33 @@
...
->如果你需要部分“重要源代码”可以提交Issues使我们进行源代码修复
-
-> If you need some "important source code", you can submit issues for us to repair the source code.
-
->如果发现其他闭源二开LiquidBounce也可以提交Issues提醒我们进行处理
-
-> If you find other closed source secondary development programs, you can also submit issues to remind us to crack them.
-
+# Bounce-Killer
+ Eliminate privately distributed/illegally sold versions of LiquidBounce. \
```
Comment (BobIsMyManager) [inline README.md:2]: Incorrect, private distribution is allowed by the GPLv3, as long as (a method to obtain) source code is distributed with it.

File: README.md [hunk 2]
```diff
@@ -1,41 +1,33 @@
...
+ Any LiquidBounce forks/modified versions that does not comply with open-source agreement will get cracked / its source leaked.
```
Comment (BobIsMyManager) [inline README.md:3]: *do not

File: README.md (hunk 1 above)
Comment (Ali00035v2) [reply README.md:2]: i thought u were dead 💀

Comment (BobIsMyManager) [reply README.md:2]: > bob i thought u were dead 💀

I told you I would be gone, but I can't stop myself from fixing things 😁.

Comment (Ali00035v2) [reply README.md:2]: > > bob i thought u were dead 💀
> 
> I told you I would be gone, but I can't stop myself from fixing things 😁.

saw you starring some repos

Comment (BobIsMyManager) [reply README.md:2]: What can I say; I'm a bit of a stargazer 😆.

**Comment by exit-scammed** [comment]
Thanks for the review, I've modified it to match your suggestions.

**Comment by RareCO** [comment]
Thank you for translating and..can you help us crack or open source?

**Comment by exit-scammed** [comment]
Unfortunately I don't have much experience in cracking, but I may try my best for a better community.

File: README.md (hunk 1 above)
Comment (Ali00035v2) [reply README.md]: > What can I say; I'm a bit of a stargazer 😆.

Same

**Pull request merged**